### ISSUE TITLE
[feature]: support jwt bearer token like postman?

### ISSUE THREAD

**Issue by kael-aiur** [hoppscotch/hoppscotch#3873]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

![image](https://github.com/hoppscotch/hoppscotch/assets/12063113/b2152a04-e9ad-4c32-b955-ef56ea85893c)

this config will be add a header to request before send request:  
```
Authorization: Bearer {jwt}
```

### Why should this be worked on?

this is oauth2 specification support jwt as access token

**Comment by MarcelHoell** [comment]
This is supported!? Just put in the JWT Baerer Token, it is working nicely on my side?

**Comment by kael-aiur** [comment]
> This is supported!? Just put in the JWT Baerer Token, it is working nicely on my side?

no, postman can fill secret an payload, then it will auto create an new jwt token with an expire time. but hoppscotch I need generate an jwt token by other way, an I must use an never expire jwt or regenerate a new one when last jwt are expired.

**Comment by liyasthomas** [comment]
Since then, we've made a lot of changes to the JWT auth method API. If you still have problems with the latest version, please let us know.
